pymc_experimental/utils/linear_cg.py

```diff
@@ -0,0 +1,279 @@
+from collections import namedtuple
+import numpy as np
+
+Setting = namedtuple("setting", "on")
+
+
+class settings:
```

Comment:
Since there are only two parameters here, could you pass `eval_cg_tolerance` and `cg_tolerance` as args to `linear_cg`?  I think it would simplify things a little bit, since you wouldn't need this class.

Reply:
This class is just a place holder, I would ideally put these two values in some sort of default settings file or class used by PyMC. I moved the constants to be global variables and kept the logic same as in GPyTorch.

Reply:
There's no need to follow GPyTorch's particular way of doing things -- in PyMC there aren't any global variables or settings files, that I'm aware of.  It's important to be consistent with these sort of things across the codebase.